### PR TITLE
KEY-57: Added option to rename the output file

### PR DESCRIPTION
**Pull request by jansensan**
per issue #4

### CODE REVIEW

**Comment by obuckley-kenzan** [comment]
Can one of the admins verify this patch?

**Comment by obuckley-kenzan** [comment]
ok to test

**Comment by obuckley-kenzan** [comment]
@jpoo90 / @jansensan 
This line has the output value hardcoded
https://github.com/kenzanmedia/pipeline-compile-less/blob/master/test/index.spec.js#L23

Should we make sure the tests cover a custom name vs. default name?  Also, I think for default, would it make sense to use the name of the current project's `package.json`?

**Comment by csharon** [comment]
@jpoo90 / @obuckley-kenzan / @jansensan 
Yes, we should have the default be the project name from the package.json.

**Comment by jansensan** [comment]
@jpoo90 @obuckley-kenzan @csharon While I agree with the logic that it should be the project name from the `package.json`, I think it's not logical to set that in the context of this project. Let me explain:

We want the resulting CSS file to have the name of the project that *includes* and *uses* pipeline-compile-less, not to be named `pipeline-compile-less.css`.

So the logic would be to have a default value set internally, like there is currently, and then have the config be made available to override, should the dev want to set the name of the resulting CSS.

Unless you see this as possible to do from within the pipeline-compile-less project, I think it's best to eave that to be done as an override.

**Comment by obuckley-kenzan** [comment]
@jansensan 
That was indeed our intent, to grab the value from the package.json of the downstream (consuming project)

**Comment by jansensan** [comment]
I'm not sure I know right now how that could be done, but if that is possible, all the better. However, I think it may be better to have that as a separate task, just so we go one improvement at a time, rather than bundling too much together?

**Comment by obuckley-kenzan** [comment]
Agreed, let me make an issue over there, then we can merge this in.

**Comment by jpoo90** [comment]
Handyman already has a function to get the package name https://github.com/kenzanmedia/pipeline-handyman#getpackagename

**Comment by jpoo90** [comment]
@jansensan @obuckley-kenzan  I can update the tests here to validate that the generated file has the name the user provided in the config.

**Comment by obuckley-kenzan** [comment]
Oh great, always good when we've already made it.  :)

**Comment by jansensan** [comment]
@obuckley-kenzan I'll use the master branch rather than my local one to see how this goes.

**Comment by jansensan** [comment]
@obuckley-kenzan So as I understand, I could easily change the line `outputFileName: 'concat.css'` to `outputFileName: handyman.getPackageName()`.

@jpoo90 When you say you cant change the test, do you mean I would resubmit my PR as is and you'll change the unit test or do you want me to change it beforehand?

**Comment by jpoo90** [comment]
@jansensan You can update the tests in your PR, that would be great. If you want me to help you doing that it isn't a big deal. 

I would use `outputFileName: handyman.getPackageName() + '.css'` , the handyman pipeline will take care of that.

**Comment by jansensan** [comment]
Changed and used `outputFileName: handyman.getPackageName() + '.css'`. Forced push branch, let's hope this PR gets updated properly.

**Comment by jansensan** [comment]
Also updated unit test that was already in place.

File: README.md [hunk 1]
```diff
@@ -51,6 +53,7 @@ Pipeline options:
   config = {
     autoprefix: true,
     concatCSS: false,
+    outputFileName: packageName.css,
```
Comment (obuckley-kenzan) [inline README.md]: should this be a string?

Comment (csharon) [inline README.md]: indeed it should, not sure if they were trying to convey that it's generated by default.

**Comment by obuckley-kenzan** [comment]
I will merge and fix in my testing PR

**Pull request merged**